### PR TITLE
fix tab order for text formatting and preview not correct

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#17328` tab order for text formatting and preview not correct`](http://community.openproject.org/work_packages/17328)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-17328-tab-order-for-text-formatting-and-preview-.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/ui_components/wiki-toolbar-directive.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ module.exports = function() {
       });
     };
 
-    element.closest('.ined-input-wrapper').find('.help').before(jQuery('<button>', previewButtonAttributes));
+    element.closest('.ined-input-wrapper').find('.help').after(jQuery('<button>', previewButtonAttributes));
```
Comment (hschink) [inline app/assets/javascripts/angular/ui_components/wiki-toolbar-directive.js]: I guess that can be easily fixed. :smile:

File: app/assets/javascripts/angular/ui_components/wiki-toolbar-directive.js [hunk 2]
```diff
@@ -51,7 +51,10 @@ module.exports = function() {
       });
     };
 
-    element.closest('.ined-input-wrapper').find('.help').before(jQuery('<button>', previewButtonAttributes));
+    element
+      .closest('.ined-input-wrapper')
+      .find('.help')
+      .after(jQuery('<button>', previewButtonAttributes));
```
Comment (0xF013) [inline app/assets/javascripts/angular/ui_components/wiki-toolbar-directive.js:57]: this should make the dog shut up

Comment (myabc) [inline app/assets/javascripts/angular/ui_components/wiki-toolbar-directive.js:57]: b7e

**Pull request merged**